docs/src/user/pytorch.rst

```diff
@@ -181,6 +190,11 @@ The maximum number of trials tried during an experiment.
 
 The maximum number of trials to be executed by a worker (a single call to ``orion hunt [...]``).
 
+``working-dir``
+
+The directory where temporary configuration files are created. If not specified, Oríon will create a
```

Comment:
They're not temporary if `--working-dir` is set though.

Reply:
Indeed!